src/poliastro/atmosphere/jacchia.py

```diff
@@ -0,0 +1,339 @@
+"""
+Given an exospheric temperature, Jacchia77 returns model
+atmospheric altitude profiles of temperature, the number
+densities of N2, O2, O, Ar, He, H, the sum thereof, and the
+molecular weight.
+
+For altitudes of 90 km and above, we use the 1977 model of
+Jacchia [Ja77].  H-atom densities are returned as non-zero
+for altitudes of 150 km and above if the maximum altitude
+requested is 500 km or more.
+
+REFERENCES:
+
+Ja77    L. G. Jacchia, "Thermospheric Temperature, Density
+        and Composition: New Models," SAO Special Report No.
+        375 (Smithsonian Institution Astrophysical
+        Observatory, Cambridge, MA, March 15, 1977).
+
+Fortran Implementation:
+https://ccmc.gsfc.nasa.gov/pub/modelweb/atmospheric/jacchia/jacchia-77/
+
+"""
+import numpy as np
+from astropy import units as u
+
+# Following constants have been taken from the fortran implementation
+pi2 = 1.57079632679
```

Comment:
I have always claimed to make use of original papers constants so final implemented model is close as possible to original one. However, I think we should use `np.pi / 2` in this case.